CMakeUserPresets.json

```diff
@@ -27,80 +27,80 @@
             "description": "Build msvc debug on windows (Developer Mode)",
-            "inherits": "build-common-debug",
+            "inherits": "common-debug",
             "configurePreset": "windows-msvc-developer"
         },
         {
-            "name": "build-windows-msvc-developer-release",
+            "name": "windows-msvc-developer-release",
             "displayName": "Release",
             "description": "Build msvc release on windows (Developer Mode)",
-            "inherits": "build-common-release",
+            "inherits": "common-release",
             "configurePreset": "windows-msvc-developer"
         },
         {
-            "name": "build-windows-msvc-user-debug",
+            "name": "windows-msvc-user-debug",
             "displayName": "Debug",
             "description": "Build msvc debug on windows (User Mode)",
-            "inherits": "build-common-debug",
+            "inherits": "common-debug",
             "configurePreset": "windows-msvc-user"
         },
         {
-            "name": "build-windows-msvc-user-release",
+            "name": "windows-msvc-user-release",
             "displayName": "Release",
             "description": "Build msvc release on windows (User Mode)",
-            "inherits": "build-common-release",
+            "inherits": "common-release",
             "configurePreset": "windows-msvc-user"
         }
     ],
     "testPresets": [
         {
-            "name": "test-windows-msvc-developer-debug",
+            "name": "windows-msvc-developer-debug",
             "displayName": "Debug",
             "description": "Set Strict rules for windows msvc (Developer Mode) debug tests",
-            "inherits": "test-common-debug",
+            "inherits": "common-debug",
             "configurePreset": "windows-msvc-developer"
         },
         {
-            "name": "test-windows-msvc-developer-release",
+            "name": "windows-msvc-developer-release",
             "displayName": "Release",
             "description": "Set Strict rules for windows msvc (Developer Mode) release tests",
-            "inherits": "test-common-release",
+            "inherits": "common-release",
             "configurePreset": "windows-msvc-developer"
         },
         {
-            "name": "test-windows-msvc-user-debug",
+            "name": "windows-msvc-user-debug",
             "displayName": "Debug",
             "description": "Set Strict rules for windows msvc (User Mode) debug tests",
-            "inherits": "test-common-debug",
+            "inherits": "common-debug",
             "configurePreset": "windows-msvc-user"
         },
         {
-            "name": "test-windows-msvc-user-release",
+            "name": "windows-msvc-user-release",
             "displayName": "Release",
             "description": "Set Strict rules for windows msvc (User Mode) release tests",
-            "inherits": "test-common-release",
+            "inherits": "common-release",
```

Comment:
I've done most of the changes you suggested in the most recent commits. If you have anything else in mind that will help future users, please comment in my PR to the upstream cpp_boilerplate_project  https://github.com/cpp-best-practices/cpp_boilerplate_project/pull/19 so we can merge it with other open cmakepresets issues we have.

Reply:
**IMHO**: to long to write on concole!

Reply:
Note: [CMake discusion](https://discourse.cmake.org/t/how-to-find-the-problem-with-an-invalid-preset-cmake-message/5390)